manifests/1.2.5/opensearch-1.2.5.yml

```diff
@@ -0,0 +1,99 @@
+---
+schema-version: '1.0'
+ci:
+  image:
+    name: opensearchstaging/ci-runner:centos7-x64-arm64-jdkmulti-node10.24.1-cypress6.9.1-20211028
+    args: -e JAVA_HOME=/opt/java/openjdk-11
+build:
+  name: OpenSearch
+  version: 1.2.5
+components:
+  - name: OpenSearch
+    repository: https://github.com/opensearch-project/OpenSearch.git
+    ref: 1.x
+    checks:
+      - gradle:publish
+      - gradle:properties:version
+  - name: common-utils
```

Comment:
I do not believe we add all components at first. Most of the components build might fail due to version incompatibility if the version is main is not incremented.